tests/my_house_tests.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+from house.constants import MINE_ADD_EXPENSE_ROUTE, MINE_EXPENSES_TITLE_ROUTE, \
+    MINE_HOUSE_TABLE_ROUTE, MINE_MINE_PAGE_ROUTE, MINE_SIDEBAR_ROUTE
+from factories.house import HouseFactory
+from house.forms import ExpenseForm
+from expenses.models import Expenses
+from django.test.client import RequestFactory
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+from tests.const import EXPENSE_AMOUNT, EXPENSE_DATE, EXPENSE_CATEGORY, EXPENSE_DESCRIPTION, EXPENSE_BAD_AMOUNT, \
+    EXPENSE_BAD_DESCRIPTION
+
+
+EXPENSE_FORM_DATA = {'date': EXPENSE_DATE, 'amount': EXPENSE_AMOUNT, 'category': EXPENSE_CATEGORY,
+                     'description': EXPENSE_DESCRIPTION}
+EXPENSE_BAD_FORM_DATA_DESCRIPTION = {'date': EXPENSE_DATE, 'amount': EXPENSE_AMOUNT, 'category': EXPENSE_CATEGORY,
+                                     'description': EXPENSE_BAD_DESCRIPTION}
+EXPENSE_BAD_FORM_DATA_AMOUNT = {'date': EXPENSE_DATE, 'amount': EXPENSE_BAD_AMOUNT, 'category': EXPENSE_CATEGORY,
+                                'description': EXPENSE_DESCRIPTION}
+
+
+@pytest.fixture
+def request_factory():
+    return RequestFactory()
```

Comment:
We are not using requstFactory anymore for this type of test, please change to use the client model
take a look at @Yoavsc129 changes or at others PR on how to do it